schemas/metadata-metaschema.yml

```diff
@@ -0,0 +1,35 @@
+$schema: "http://json-schema.org/draft-06/schema#"
```

Comment:
how about:
```
$schema: "https://taskcluster.net/schemas/common/metadata-metaschema.json#"
metadata:
  - name: "metadata-metaschema"
  - version: 1
```

;-)

Reply:
It seems like having a document be its own metaschema could lead to trouble?  What's the benefit here?

Reply:
Or maybe this even works(?):

```
$schema: /#
```